arm/ui/templates/base.html

```diff
@@ -10,9 +10,7 @@
             {% with messages = get_flashed_messages(with_categories=true) %}
                 {% if messages %}
                     {% for category, message in messages %}
-                        <div id="message1" class="alert alert-{{ category if category !=" message
-            " else 'info' }} text-center"
-                             role="alert">
+                        <div id="message1" class="alert alert-{{ category if category != "message" else 'info' }} text-center" role="alert">
```

Comment:
See comment about flash. These are no longer used as it's better to use the notification system.

Reply:
This change was to remove a carriage return that might be the cause of problems when rendering the page.